commands/reboot.js

```diff
@@ -1,9 +1,9 @@
-exports.run = async (client, message, args, level) => {// eslint-disable-line no-unused-vars
+exports.run = async (client, message, args, level) => { // eslint-disable-line no-unused-vars
   await message.reply("Bot is shutting down.");
-  client.commands.forEach( async cmd => {
-    await client.unloadCommand(cmd);
-  });
-  process.exit(1);
+  await Promise.all(client.commands.map(cmd =>
```

Comment:
Please explain the difference between these lines.

Reply:
I just changed thinking, the master one is updated version